### PR TITLE
Adds instructions for Windows and upgrades client library versions.

### PR DESCRIPTION
**Pull request by gguuss**

### CODE REVIEW

File: video/cloud-client/README.md [hunk 1]
```diff
@@ -59,3 +59,12 @@ Detect Shots
...
     com.example.video.Detect shots gs://cloudmleap/video/next/gbikes_dinosaur.mp4
 ```
+
+From Windows, you may need to supply your classpath diferently, for example:
+```
+java -cp target\\video-google-cloud-samples-1.0.0-jar-with-dependencies.jar com.example.video.Detect labels gs://demomaker/cat.mp4
+```
+or
+```
+java -cp target\\video-google-cloud-samples-1.0.0-jar-with-dependencies.jar com.example.video.Detect labels-file resources/cat.mp4
```
Comment (jabubake) [inline video/cloud-client/README.md]: should n't 'resources/cat.mp4' also be 'resources\\cat.mp4' ?
I'm concerned of maintenance / consistency of these comments across our samples : is there a general statement we can make in our samples that paths to local directories are made in Unix style and need to be updated to the backslash for Windows environments ?

Comment (gguuss) [inline video/cloud-client/README.md]: Great point!  We're using the PATH module which localizes this aspect of the file path to the correct path though so you don't need to escape this part of the path.

Comment (gguuss) [inline video/cloud-client/README.md]: I left the path with the forward slashes there on purpose for consistency but am happy to make the change if you feel it's better with escaped \\s.

**Comment by gguuss** [comment]
@lesv @jabubake I'm going to go ahead and merge, this hopefully will help a customer I'm working with.

**Comment by lesv** [comment]
LGTM

**Pull request merged**